src/app/docs/icon/icon-docs.component.html

```diff
@@ -4,143 +4,170 @@
-        size="3x"
-      >
-      </sky-icon>
-      <sky-icon
-        icon="trash"
-        size="sm"
-      >
-      </sky-icon>
-      <sky-icon
-        icon="users"
-        size="4x"
-      >
-      </sky-icon>
-      <sky-icon
-        icon="user"
-        size="5x"
-      >
-      </sky-icon>
-      <sky-icon
-        icon="sort"
-        iconType="skyux"
-        size="5x"
-      >
-      </sky-icon>
-    </div>
-  </sky-docs-demo>
-
 
   <sky-docs-design-guidelines>
 
     <sky-docs-demo-page-section
-      heading="Usage"
+      heading="Principles"
     >
-      <p>
-        Icons are pictorial representations of objects. They help define the visual identity of SKY UX applications, and they provide a shorthand to visually convey meaning that users can understand at a glance. SKY UX uses glyphic, mono-color icons to facilitate, guide, and inform users.
+      <p class="sky-docs-text-block">
+        Icons should be used deliberately to facilitate user comprehension and drive focus to an action or section; moreover, they should be:
       </p>
-      <p>
-        Icons should not be used solely for decorative purposes.
+      <ul>
+        <li>
+          <p class="sky-docs-text-block">
+            <b>Global: </b>Use established, universally-known icons in a purposeful manner wherever possible
+          </p>
+        </li>
+        <li>
+          <p class="sky-docs-text-block">
+            <b>Supplemental: </b>Icons should be paired with text wherever possible; avoid using icons solely for decoration or visual interest
+          </p>
+        </li>
+        <li>
+          <p class="sky-docs-text-block">
+            <b>Specific: </b>Icons should have a single meaning, and variations should be limited: A single icon should represent a common idea or subject and should not be abstract.
+          </p>
+        </li>
+      </ul>
+
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Icon styling"
+    >
+      <p class="sky-docs-text-block">
+        SKY UX uses simple, glyphic, mono-color icons to facilitate, guide, and inform users. Icons are designed in Blackbaud's brand personality, reflecting a human, visionary approach, and are built on the SKY UX's modern theme's visual language.
       </p>
 
       <sky-docs-design-guideline
-        heading="How to select icons"
+        heading="Color"
       >
-        <p>
-          To determine whether an icon is available for a new component, check the SKY UX icon dictionary below. Then make sure that your use case is consistent with how other components use the icon.
+        <p class="sky-docs-text-block">
+          Icons are always a solid, monochromatic color and must meet the <a href="https://www.w3.org/WAI/WCAG21/Techniques/general/G207.html" target="_blank">WCAG contrast ratio of 3:1</a>. An icon's color should be reflective of the component to which it is tied. For more information, view the SKY UX color guidelines for modern theme.
```

Comment:
We mention modern theme in this paragraph. Should we use the `if` property so that this only appears in the modern theme docs?

Reply:
Great catch - yes, I'll update